### ISSUE TITLE
[BUG] Gitako extension not working with TWP

### ISSUE THREAD

**Issue by boromyr** [FilipePS/Traduzir-paginas-web#570]
TWP machine translation appears to block the use of the Gitako extension on GitHub.com as seen in the attached gif. The problem also occurs after resetting the TWP settings to their defaults, and only with machine translation.
![TWP bug](https://user-images.githubusercontent.com/52782032/212983541-cca026ee-7307-4df7-872e-2a982ffc025d.gif)

To reproduce the problem just install https://chrome.google.com/webstore/detail/gitako/giljefjcheohhamkjphiebfjnlphnokk, login with , give Gitako permission to access GitHub via OAuth or token, and enable machine translation with TWP.

Windows 11 
Edge stable 109

**Comment by FilipePS** [comment]
Is this problem still happening?

**Comment by boromyr** [comment]
Yes, TWP version 9.8 (dev) commit https://github.com/FilipePS/Traduzir-paginas-web/commit/eb24d7847a54bce7dfd792bf1f3f8c7d1853d560

**Comment by FilipePS** [comment]
Please confirm that the last commit really solved this problem

**Comment by boromyr** [comment]
I'm sorry, unfortunately the problem persists. Has the problem been solved in your tests? In case I'll do a test by resetting all the settings.

**Comment by FilipePS** [comment]
Are you sure you tested the correct version?
Test this one:
[TWP_9974d69_Chromium.zip](https://github.com/FilipePS/Traduzir-paginas-web/files/11472692/TWP_9974d69_Chromium.zip)
